api/src/db-connection.ts

```diff
@@ -93,6 +79,7 @@ export class Connection {
    LEFT JOIN entity
           ON entity.paper_id = paper.s2_id
          AND entity.version = version.index
+         AND entity.type = '${entity_type}'
```

Comment:
Would be nice to see this using knex's parameter substitution instead of inserting the string directly into the SQL query. There's an example in [getPaperEntityCount](https://github.com/allenai/scholarphi/blob/master/api/src/db-connection.ts#L118-L140)